src/control/conf.h

```diff
@@ -215,7 +215,7 @@ static inline void dt_conf_init(dt_conf_t *cf, const char *filename, GSList *ove
     if(!i)
     {
       snprintf(darktable.conf->filename, sizeof(darktable.conf->filename), "%s", filename);
-      f = fopen(filename, "rb");
+      f = fopen(filename, "r");
```

Comment:
Hi @houz, yes, that is also working. However the resulting `darktablerc` file has now always unix style endings (`\n`) upon program exit, I guess because of [this](https://github.com/darktable-org/darktable/blob/master/src/control/conf.h#L272). I don't mind as my editor can handle both line endings, but default Notepad would be challanged. Are you OK with that or should we find a solution?